buildSrc/src/main/kotlin/OneSkyConfiguration.kt

```diff
@@ -0,0 +1,47 @@
+import co.brainly.onesky.OneSkyPluginExtension
+import co.brainly.onesky.task.DEPRECATE_STRINGS_FLAG
+import co.brainly.onesky.task.DownloadTranslationsTask
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.apply
+import org.gradle.kotlin.dsl.configure
+import org.gradle.kotlin.dsl.extra
+
+private const val PROP_API_KEY = "ONESKY_API_KEY"
+private const val PROP_API_SECRET = "ONESKY_API_SECRET"
+
+fun Project.onesky(configuration: OneSkyPluginExtension.() -> Unit) {
+    extra.set(DEPRECATE_STRINGS_FLAG, true)
+    apply(plugin = "co.brainly.onesky")
+
+    configure<OneSkyPluginExtension> {
+        apiKey = findProperty(PROP_API_KEY)?.toString().orEmpty()
+        apiSecret = findProperty(PROP_API_SECRET)?.toString().orEmpty()
```

Comment:
@gyasistory here is how you can use parameters without needing to define a placeholder value in the `gradle.properties` file